monkey/infection_monkey/command_builders/utils.py

```diff
@@ -8,16 +8,31 @@
 
 
 def get_agent_argument(run_options: Union[WindowsRunOptions, LinuxRunOptions]) -> str:
+    """
+    Get the appropriate agent argument based on the run options.
+
+    :param run_options: The run command options.
+    :raises ValueError: If dropper execution mode is DropperExecutionMode.SCRIPT
+    :return: The Agent argument to be used.
+    """
     agent_arg = MONKEY_ARG
     if run_options.dropper_execution_mode == DropperExecutionMode.DROPPER:
         agent_arg = DROPPER_ARG
+    if run_options.dropper_execution_mode == DropperExecutionMode.SCRIPT:
+        raise ValueError("Dropper execution mode should not be `DropperExecutionMode.Script`")
```

Comment:
What happens if we add a new `DropperExecutionMode`. Will we remember to update the logic in this function?

Reply:
Probably not. I will change the logic.